README.md

```diff
@@ -41,31 +41,70 @@ Note, if you are using the docker MySQL database, ensure that in `src/main/resou
 ## Development
 ### Pre-requisites
 #### Node/Yarn
-The frontend is built using Vue.js. To run the frontend, you will need to have `yarn`. After installing `Node`, run the following to enable `yarn` globally:
+The frontend is built using Vue.js. To run the frontend, you will need to have `yarn`.
+After installing `Node`, run the following to enable `yarn` globally:
 ```bash
 corepack enable
 ```
 (`sudo` may be required)
 
-The frontend can be run in development mode by running the following in the `src/main/resources/frontend` directory:
-```bash
-yarn dev
-```
+#### Backend
+A local installation of maven is required to run the backend.
+
+Alternatively, you can run the database locally with your own MySQL server.
+Be sure to update `src/main/resources/db.properties` with the correct database url, username, and password.
+
+#### Database
+To run the database locally, you will need to have `docker` installed.
 
-#### Docker
-To run the database locally, you will need to have `docker` installed. Run the following in the root of the project to start the db container:
+Alternatively, you can run the database locally with your own MySQL server.
+Be sure to update `src/main/resources/db.properties` with the correct database url, username, and password.
+
+##### Docker
+Run the following in the root of the project to start the db container:
 ```bash
 docker compose up db -d
 ```
 
-#### Maven
-A local installation of maven is required to run the backend.
+##### MySQL
+Go fish🐟
 
+_This is required, but no setup instructions are given in this document._
 
-Alternatively, you can run the database locally with your own MySQL server. Be sure to update `src/main/resources/db.properties` with the correct database url, username, and password.
+### Getting Started
+NOTE: These instructions will help you set up this project in Intelli-J.
```

Comment:
How do we feel about specializing on **Intelli-J**? Do we want to be more general and support VSCode as well? Or, like the attitude we're taking in the class, are we OK with helping people adopt the system of _this_ organization which is Intelli-J? (That's the approach I assume we're taking, and I assume we're ok with it. Furthermore, it's the only information I have at the this point.)

Reply:
I think it's fine to expect IntelliJ. I do not think it would be very difficult for somebody to adopt this to work for VSCode or some other platform